src/get-pokemons.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Try to resolve instead of disabling

Reply:
This issue is well addressed. Good job! :clap: :+1: